src/utils/is-url.js

```diff
@@ -4,7 +4,7 @@ const isURL = require('is-url');
 const ANCHOR_REGEXP = /^#/;
 
 // Matches scheme (ie: tel:, mailto:, data:)
-const SCHEME_REGEXP = /^[a-z]*\:/i;
+const SCHEME_REGEXP = /^[a-z]*:/i;
```

Comment:
Here too. ESLint seems to have removed some bit of regex. Again, ignore if I'm wrong. 😅

Reply:
Same here. The \ was an unnecessary character escape. Same functionality.